projects/demo/src/modules/animations/examples/fade-in/index.ts

```diff
@@ -25,10 +25,7 @@ export class TuiFadeInExample {
     );
 
     @tuiPure
-    getAnimation(duration: number) {
-        return {
-            value: '',
-            params: {duration},
-        } as const;
+    getAnimation(duration: number): TuiDurationOptions {
+        return {value: '', params: {duration}} as const;
```

Comment:
do we need `as const` if we have return-type ?

Reply:
fixed